packages/dialtone-icons/src/keywords.json

```diff
@@ -519,6 +520,12 @@
         "emoji",
         "activity"
       ],
+      "gif-fill": [
+        "giphy"
+      ],
```

Comment:
@yorbi-dp,

Marshall changed this to just `gif`, removing the `-fill`. Also he hadn't yet merged the Figma branch, even though I approved him to do so. Go ahead and merge it.

Meanwhile, I think there's an outstanding task to change all of the icons to `color/foreground/primary`. He might not have gotten to it, but that should definitely be its own branch.

Reply:
Thanks for the heads up! Fixed in the lastest commit.

Just merged the branch. Will do the change of colors in a new one!